src/hooks/renderTarget.ts

```diff
@@ -15,3 +16,22 @@ export const useRenderTarget = (width = 0, height = 0) => {
     [width, height, renderTargetSize]
   );
 };
+
+export const useRenderTargets = (
+  layer: LayerV2,
+  width?: number,
+  height?: number
+) => {
+  const { renderTargetSize } = useStore().uiStore;
+  return useMemo(() => {
+    const numRenderTargets = layer.maxConcurrentBlocks + 1;
+    return Array.from(
+      { length: numRenderTargets },
+      (_) =>
+        new WebGLRenderTarget(
+          width || renderTargetSize,
+          height || renderTargetSize
+        )
+    );
+  }, [layer, width, height, renderTargetSize]);
```

Comment:
We probably want to include `layer.maxConcurrentBlocks`

Reply:
Is it not enough to just include `layer`? Or are you saying we should narrow that down to just `layer.maxConcurrentBlocks` to avoid unnecessary re-executions of the hook?

Reply:
It won't be enough to include layer, because the layer instance doesn't change after experience deserialization. We mutate the layer instance properties, but its reference remains constant. Since we want it to specifically react to changes in maxConcurrentBlocks, we'll have to include that here